typescript/deploy/src/utils.ts

```diff
@@ -9,12 +9,13 @@ export function getArgs() {
   return yargs(process.argv.slice(2))
     .alias('e', 'env')
     .describe('e', 'deploy environment')
+    .string('e')
```

Comment:
I think @jmrossy was removing yargs IIRC? Should we move this into infra?

Reply:
Yep, removed from deploy/SDK in my PR and moved to infra